lms/static/sass/elements/_controls.scss

```diff
@@ -323,3 +323,118 @@
       margin-bottom: none;
     }
   }
+
+
+// ===================
+// Dropdown menus
+// ===================
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    position: relative;
+    display: inline-block;
+
+    .button-more.has-dropdown {
+        @extend %ui-depth2;
+        @include transition(all, $tmg-f2);
+        position: relative;
+        padding: ($baseline/2);
+        border: 1px solid transparent;
+        background: $white;
+        box-shadow: none;
+        font-size: $body-font-size;
+        text-shadow: none; // Clears text shadows on all buttons
+        line-height: ($baseline/2);
+
+        .icon {
+            @include transition(all, $tmg-f2);
+            @include transform-origin(45% center); // odd first value to keep icons centered
+            color: $gray-l1;
+        }
+
+        &:hover,
+        &:focus {
+            border-color: $gray-l3;
+            background: $white;
+
+            .icon {
+                color: $gray-d3;
+            }
+        }
+
+        &.is-active {
+            border-bottom-width: 0;
+            border-bottom-left-radius: 0;
+            border-bottom-right-radius: 0;
+            border-color: $gray-l3;
+            background: $white;
+
+            .icon {
+                @include transform(rotate(-180deg));
+                color: $gray-d3;
+            }
+        }
+
+        & + .dropdown-menu {
+            @extend %ui-depth1;
+            @extend %ui-no-list;
+            @extend %anim-fadeIn;
+            position: absolute;
+            min-width: ($baseline*7.5);
+            max-width: ($baseline*12.5);
+            margin-top: -1px;
+            border: 1px solid $gray-l3;
+            border-radius: 3px;
+            border-top-left-radius: 0;
+            border-top-right-radius: 0;
+            background: $white;
+            box-shadow: 0 2px 2px rgba(0, 0, 0, .15);
+            font-size: $body-font-size;
+            text-indent: 0;
+
+            &.is-hidden {
+                @extend %text-sr;
+            }
+
+            &.align-left {
+                left: 0;
+
+                &.control-text-align {
+                    @include text-align(left);
+                }
+            }
+
+            &.align-right {
+                right: 0;
```

Comment:
We actually don't want RTL here, as it works RTL the way it is. I originally had RTL here, but the positioning was backwards.